src/inputs/dropdownMenu/dropdownMenu.jsx

```diff
@@ -224,7 +224,7 @@ function DropdownMenu(props) {
     useEffect(() => {
         function onClickOutside(event) {
             const parentContainer = dropdownMenuRef.current;
-            const containsTarget = parentContainer.contains(event.target);
+            const containsTarget = !isNil(parentContainer) && parentContainer.contains(event.target);
```

Comment:
Just an idea:

We can minimise the dependency on Lodash, we can do:

```parentContainer?.contains?.(event.target)```

In any case the returned value would be `undefined` or `null` if `parentContainer` is empty.

Reply:
@ajaybushel Yes we can, but it required installing additional Babel plugins not currently installed here, and we were in a little bit of a hurry with this release.  But just so you know I had the same thought, see 37bac18 and then 84d72d7.  GMTA! =)